mods/d2k/rules/aircraft.yaml

```diff
@@ -1,46 +1,98 @@
-^CARRYALL:
+carryall.scripted:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 1200
 	Tooltip:
 		Name: Carryall
 		Description: Large winged, planet-bound ship\n  Automatically lifts harvesters.
-	Buildable:
-		Queue: Aircraft
-		BuildPaletteOrder: 120
 	Health:
 		HP: 250
 	Armor:
 		Type: Light
 	Helicopter:
-		CruiseAltitude: 2100
+		CruiseAltitude: 2048
```

Comment:
This will probably want to use the values for the default carryall. There should be no reason to use different values.